osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -102,6 +102,14 @@ public BeatmapOnlineStatus Status
 
         public string OnlineMD5Hash { get; set; } = string.Empty;
 
+        /// <summary>
+        /// The last time of a local modification (via the editor).
+        /// </summary>
+        public DateTimeOffset? LastUpdated { get; set; }
+
+        /// <summary>
+        /// The last time online metadata was applied to this beatmap.
+        /// </summary>
         public DateTimeOffset? LastOnlineUpdate { get; set; }
```

Comment:
I'm not sure how to feel about having both `DateTimeOffset? BeatmapInfo.LastUpdated` (added here) and `DateTimeOffset? BeatmapInfo.LastOnlineUpdate` (already present). Can we rename the former to something like `LastLocalUpdate` or something for clarity?

Reply:
Looking back at this, I'm wondering if we can just have a single property to handle both? I can't think of a case where we'd use the online value if there are local modifications. And it would be reset to the correct value the next time an update is applied (restoring to a non-modified state).

Reply:
Maaaybe? I'm not sure I have a well formed opinion on that. `LastOnlineUpdate` is being used in a [few](https://github.com/ppy/osu/blob/2205e0dc3dde02fa20f13824b147d6787a582ea4/osu.Game/BackgroundBeatmapProcessor.cs#L105) [places](https://github.com/ppy/osu/blob/f9c6dc314b083ddc96f071de38da57940f52413e/osu.Game/Beatmaps/BeatmapInfo.cs#L111) that have online-specific checks, so I'm not sure merging is possible on technical merit, but I'm not very familiar with the beatmap update flow anymore.

Reply:
Yeah, best to keep separate for now. We can complicate things later if required.

I'll rename to `LastLocalUpdate` for now.